engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -67,6 +68,23 @@ public void setReason(String reason)
     this.reason = reason;
   }
 
+  public void setLogLevel(LogLevel logLevel)
```

Comment:
Currently, the log level associated with each type of StramEvent is static. However, this may not be the case always, the event may depend on the context in which it is raised, for eg, StopContainerEvent may be raised either because 
a) some issue in the container e.g. insufficient memory
b) the user himself may have requested a container kill.
Case (a) should generate an event with log level ERROR whereas case (b) should generate event with log level INFO.
Do you suggest adding/changing the constructor for each StramEvent instead of setter method?